### PR TITLE
Featutre sb 61 otp email verifying

### PR DESCRIPTION
**Pull request by jaroslawtt**
image

### CODE REVIEW

**Comment by lyudad** [comment]
I think code inputs are too wide

File: src/libs/hooks/use-timer.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+};
+
+type ReturnFunctionType = {
+	resetTimer: () => void;
+	startTimer: () => void;
+	stopTimer: () => void;
+	timeRemaining: ReadonlySignal<string>;
+};
+
+const useTimer = ({ min, sec }: Options): ReturnFunctionType => {
```
Comment (lyudad) [inline src/libs/hooks/use-timer.ts:22]: comment needed what does this function do

Comment (jaroslawtt) [reply src/libs/hooks/use-timer.ts:22]: hook for timer, that calculates time (before code expires for instance)

**Comment by jaroslawtt** [comment]
> I think code inputs are too wide

image
reduced size

**Pull request merged**